src/fstests/fstests.jinja2

```diff
@@ -63,7 +109,9 @@ class Job(BaseJob):
         if self._check_kvm_xfstest_env():
             if self._kernel_config(src_path) == 'pass':
                 if self._kernel_build(src_path) == 'pass':
-                    return self._run_smoke_tests()
-        else:
-            return 'fail'
+                    if self._run_smoke_tests() == 'pass':
```

Comment:
We could have some code like `def _run_fstests(self, src_path):` which does:
```python
if not self._kernel_config(src_path):
    return False
if not self._kernel_build(src_path):
    return False
# etc.
```
Then return `'pass' if res is True else 'fail'` from `_run()`.

Reply:
Updated accordingly